### ISSUE TITLE
Location autocomplete

### ISSUE THREAD

**Issue by vict0rsch** [cc-ai/climatecollection#9]
Maybe add an option to apply to all images in current upload as they are likely to come from the same location

to be discussed

**Comment by Vahe987** [comment]
good idea, but this will create difficulty from user point of view I think
fyi, we are going to have a meeting with UX designer, will discuss with her

**Comment by vict0rsch** [comment]
yeah, just a suggestion to discuss :)

**Comment by Vahe987** [comment]
cool, will ask her, thank you
